lib/vanagon/platform/rpm.rb

```diff
@@ -69,6 +69,7 @@ def initialize(name)
           @install = "/opt/freeware/bin/install"
         end
         @rpmbuild = "/usr/bin/rpmbuild"
+        @bash = "/bin/bash"
```

Comment:
I want to wait until Haus' other pull request refactoring this is merged, and then we'll have separate definitions for aix, rpm, etc.